include/RAJA/util/View.hpp

```diff
@@ -162,6 +162,11 @@ struct MultiView
   layout_type const layout;
   nc_pointer_type data;
```

Comment:
data should be pointer_type.

Need constructors for pointer_type is const T ** from non-const data.